### DISCUSSION TITLE
cant get helix-term v0.6.0

### DISCUSSION THREAD

**Discussion by Caeser-SONG** [helix-editor/helix#3736] [General]
helix-term v0.6.0  cant  build  

how can i get it

error: failed to compile `helix-term v0.6.0 (/base/helix/helix-term)`, intermediate artifacts can be found at `base/helix/target`

**Comment by kirawi** [comment]
It's unclear with the information provided. It should have explained why it failed to compile along with the message (could you provide the entire message?). You can also just download directly from here: https://github.com/helix-editor/helix/releases/tag/22.08.1

**Reply by Caeser-SONG** [reply]
thanks:)

![image](https://user-images.githubusercontent.com/50317049/189023449-c2455a12-79d1-4d4d-a88d-cbedd1ad73fb.png)

I execute this command `cargo install --path helix-term`  to build helix from source. but

I stop at `Building [=======================> ] 205/208: helix-term(build)`

**Reply by kirawi** [reply]
Is there a message after `Building [=======================> ] 205/208: helix-term(build)`? Or is `failed to compile` the entire thing?
